src/as_execution/abi.rs

```diff
@@ -863,13 +872,162 @@ pub fn assembly_script_function_exists(
     Ok(function_exists(&mut ctx, address, function)? as i32)
 }
 
+/// Called by the instance when an error popped. It print the filename where the error
```

Comment:
Maybe we can change the first phrase as it is not very informative?

Reply:
Done